src/control/externprogram.cpp

```diff
@@ -1,11 +1,12 @@
 /*!
-	Copyright (c) 2006-2008, Reinhard Katzmann, Matevž Jekovec, Canorus development team
+	Copyright (c) 2006-2019, Reinhard Katzmann, Matevž Jekovec, Canorus development team
```

Comment:
This should be 2020 now ;)

Reply:
However I did change this file in 2019...